src/app/_components/DistrictSearch/DistrictSearch.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import { cn } from '@utils/cn';
```

Comment:
🟢 it is also accessible in @utils i think

Reply:
![image](https://github.com/keenethics/mayak/assets/103147058/ad229b64-3f77-48d1-8620-7e3f533376f6)